src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/your-first-nrql-condition.mdx

```diff
@@ -10,127 +10,278 @@ redirects:
 
-Learn more about the [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#syntax) to create other useful queries.
+  Next, you need to set the priority level. The priority level determines what will create an incident. We recommend setting your priority level as **critical** for your first alert. You can learn more about priority levels in our [alert condition docs](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/advanced-techniques/set-thresholds-alert-condition/#threshold-levels).
 
-### Step 2: Name your NRQL condition [#step2-nrql]
+  Next, you will need to define what defines a **critical anomaly threshold violation**. For this first alert, we recommend using our default settings and adjusting to your needs as necessary. So, leave the settings to open an incident "when a query returns a value deviates from the predicted value: **for at least five minutes** by **3 standard deviations**".
 
-Give your NRQL condition a name that's meaningful to you.
+  Learn more about anomalies in our anomaly (documentation)[/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/anomaly-detection-applied-intelligence/].
 
-### Step 3: Set your condition thresholds [#step3-nrql]
+  <img
+    title="Configure your anomaly thresholds"
+    alt="A screenshot demonstrating how to configure custom anomalies"
+    src={alertsConfigureYourAnomalyThresholds}
+  />
 
-Set critical and warning thresholds to determine when you're notified about your environment's performance. You must configure at least one threshold.
+</Collapser>
 
-* A critical threshold is optional for your alerts condition.  
+  <Collapser
+    id="static-threshold"
+    title="Static threshold"
+  >
+    Unlike anomaly thresholds, a static threshold doesn't look at your data set as a whole and determine what behavior is unusual based on your systems history, instead a static threshold will open an incident whenever your system behaves differently than the conditions that **you set**. Static alert thresholds are much more customizable and we recommend them if you've a strong sense of your data and what you're looking for. 
```

Comment:
This is a bit confusing to me, I suspect because it's a sentence stating what static thresholds don't do. I'm not sure how to fix it, but the area that trips me up is "data set as a whole and determine". Also I think the comma can be a period and you can start a new sentence with "Instead, a static threshold," That may help a bit.

Reply:
done!